config/sync/shortcut.set.default.yml

```diff
@@ -1,4 +1,4 @@
-uuid: NULL
+uuid: d33bd571-9d49-41d0-9713-fa09d806c25c
```

Comment:
FYI: This re-introduces the core shortcut bug. It's best to not commit this. In general, only commit files that are relevant to what you were working on.

Reply:
I missed this. In one of the classes could we have a discussion of adding files to .gitignore? I looked it up, but there are several options for where to ask git to ignore files depending on the use case.